Content.Client/GameObjects/Components/PDA/PDABoundUserInterface.cs

```diff
@@ -106,19 +106,32 @@ protected override void UpdateState(BoundUserInterfaceState state)
                             msg.PDAOwnerInfo.ActualOwnerName));
                     }
 
-                    if (msg.PDAOwnerInfo.JobTitle == null || msg.PDAOwnerInfo.IdOwner == null)
-                    {
-                        _menu.IDInfoLabel.SetMarkup(Loc.GetString("ID:"));
-                    }
-                    else
+                    if (msg.PDAOwnerInfo.JobTitle != null && msg.PDAOwnerInfo.IdOwner != null)
                     {
                         _menu.IDInfoLabel.SetMarkup(Loc.GetString(
                             "ID: [color=white]{0}[/color], [color=yellow]{1}[/color]",
                             msg.PDAOwnerInfo.IdOwner,
+                            msg.PDAOwnerInfo.JobTitle?? ""));
```

Comment:
you shouldn't need the null coalesce since you checked that JobTitle isn't null already